spec/helpers/ruby_version_spec.rb

```diff
@@ -117,32 +131,42 @@
     end
   end
 
-
   it "detects non mri engines" do
-    Hatchet::App.new("ruby_193_jruby_173").in_directory_fork do |dir|
+    Hatchet::App.new("default_ruby").in_directory_fork do |_|
+      dir = Pathname(Dir.pwd)
       Bundler.with_unbundled_env do
-        ruby_version   = LanguagePack::RubyVersion.new(@bundler.install.ruby_version, is_new: true)
-        version_number = "1.9.3"
-        engine_version = "1.7.3"
-        engine         = :jruby
-        version        = "ruby-#{version_number}-#{engine}-#{engine_version}"
-        to_gemfile     = "ruby '#{version_number}', :engine => '#{engine}', :engine_version => '#{engine_version}'"
-        expect(ruby_version.version_without_patchlevel).to eq(version)
+        dir.join("Gemfile").write(<<~EOF)
+          source "https://rubygems.org"
+
+          ruby '2.6.8', engine: 'jruby', engine_version: '9.3.6.0'
+        EOF
+        dir.join("Gemfile.lock").write(<<~EOF)
+          GEM
+            remote: https://rubygems.org/
+            specs:
+          PLATFORMS
+            java
+
+          DEPENDENCIES
+
+          RUBY VERSION
+            ruby 2.6.8p001 (jruby 9.3.6.0)
+
+          BUNDLED WITH
+            2.3.25
+        EOF
+
+        ruby_version   = LanguagePack::RubyVersion.new(
+          @bundler.install.ruby_version,
+          is_new: true
+        )
+        version_number = "2.6.8"
+        engine_version = "9.3.6.0"
+        engine = :jruby
+        expect(ruby_version.version_without_patchlevel).to eq("ruby-#{version_number}-#{engine}-#{engine_version}")
         expect(ruby_version.engine_version).to eq(engine_version)
-        expect(ruby_version.to_gemfile).to eq(to_gemfile)
         expect(ruby_version.engine).to eq(engine)
       end
     end
   end
-
-  it "surfaces error message from bundler"  do
-    bundle_error_msg = "Zomg der was a problem in da gemfile"
-    error_klass      = LanguagePack::Helpers::BundlerWrapper::GemfileParseError
-    Hatchet::App.new("bad_gemfile_on_platform").in_directory_fork do |dir|
-      Bundler.with_unbundled_env do
-        @bundler       = LanguagePack::Helpers::BundlerWrapper.new().install
-        expect { LanguagePack::RubyVersion.new(@bundler.ruby_version) }.to raise_error(error_klass, /#{Regexp.escape(bundle_error_msg)}/)
-      end
-    end
-  end
```

Comment:
This test is supposed to exercise this code https://github.com/heroku/heroku-buildpack-ruby/blob/7866d1efbdc0e4d5f66caff2cdd7f83f76d4f8e0/lib/language_pack/helpers/bundler_wrapper.rb#L176

However bundler no longer errors on `bundle platform --ruby` depending on what version of bundler is running.

In the future we will move to get rid of the concept of "blessed" bundler versions and to BYOBv (bring your own bundler version) which means all that code will eventually go away. I don't want to remove the check and re-raise logic right now though. If we wanted to test this functionality we would need to move away from these local "unit-ish" tests and to a full blown hatchet test that tries to deploy an app.